.github/workflows/report.yml

```diff
@@ -114,12 +157,24 @@ jobs:
           path: site/
           merge-multiple: true
 
-      - name: Include Implementation Metadata
+      - name: Include Implementations Metadata (Private API)
         uses: actions/download-artifact@v4
         with:
           name: implementations
           path: site/
 
+      - name: Create a Public Directory Structure for JSON Schema Org
+        run: mkdir -p site/v1/json-schema-org
```

Comment:
Possibly you're missing an `api/`? But can't you just make stuff above output the right directory structure so that there's no need to rename things here?

Reply:
Still working on the PR 😅. Was going to ping you once pushing the commits was done.

Reply:
> But can't you just make stuff above output the right directory structure so that there's no need to rename things here?

Have to do the renaming because we want this to be accessible --> https://bowtie.report/api/v1/json-schema-org/implementations and not https://bowtie.report/api/v1/json-schema-org/implementations-file-for-json-schema-org

At this line --> https://github.com/bowtie-json-schema/bowtie/blob/fa9f61e4c47aba3eaea1eaa8bd7c8f1100768bd9/.github/workflows/report.yml#L91

there's already an `implementations` artifact getting created for our private API usage so again a same named artifact can't be created as it will cause conflict while downloading.

Reply:
So I have to essentially create a different named file and then download it and rename it just to avoid conflict. Can't think of a workaround right now for this. Will come back to it if i find something.

Reply:
You'd create an artifact called `api` instead, and put *all* of the nested files (exactly 1 for now) in it :)!

Reply:
Right! Thanks :)